sql/core/src/test/scala/org/apache/spark/sql/FilterPushdownBenchmark.scala

```diff
@@ -0,0 +1,195 @@
+        Parquet Vectorized                            2085 / 2165          0.5        1988.0       1.0X
+        Parquet Vectorized (Pushdown)                  704 /  769          1.5         671.1       3.0X
+        Native ORC Vectorized                         1637 / 1638          0.6        1561.1       1.3X
+        Native ORC Vectorized (Pushdown)                76 /   79         13.8          72.4      27.4X
+
+        Filter Pushdown (id < 1):                Best/Avg Time(ms)    Rate(M/s)   Per Row(ns)   Relative
+        ------------------------------------------------------------------------------------------------
+        Parquet Vectorized                            2069 / 2133          0.5        1972.7       1.0X
+        Parquet Vectorized (Pushdown)                  705 /  764          1.5         672.7       2.9X
+        Native ORC Vectorized                         1637 / 1651          0.6        1561.3       1.3X
+        Native ORC Vectorized (Pushdown)                75 /   77         14.0          71.4      27.6X
+
+        Filter Pushdown (id IS NULL):            Best/Avg Time(ms)    Rate(M/s)   Per Row(ns)   Relative
+        ------------------------------------------------------------------------------------------------
+        Parquet Vectorized                            2081 / 2123          0.5        1984.4       1.0X
+        Parquet Vectorized (Pushdown)                   36 /   37         29.3          34.1      58.1X
+        Native ORC Vectorized                         1616 / 1645          0.6        1540.7       1.3X
+        Native ORC Vectorized (Pushdown)                41 /   43         25.7          39.0      50.9X
+        */
+
+        // Negative cases: Select all rows which means the predicate is always true.
+        /*
+        Java HotSpot(TM) 64-Bit Server VM 1.8.0_152-b16 on Mac OS X 10.13.2
+        Intel(R) Core(TM) i7-4770HQ CPU @ 2.20GHz
+
+        Filter Pushdown (id > -1):               Best/Avg Time(ms)    Rate(M/s)   Per Row(ns)   Relative
+        ------------------------------------------------------------------------------------------------
+        Parquet Vectorized                            8346 / 8516          0.1        7959.8       1.0X
+        Parquet Vectorized (Pushdown)                 8611 / 8630          0.1        8212.4       1.0X
+        Native ORC Vectorized                         7700 / 7940          0.1        7343.2       1.1X
+        Native ORC Vectorized (Pushdown)              7572 / 7635          0.1        7221.5       1.1X
+
+        Filter Pushdown (id != -1):              Best/Avg Time(ms)    Rate(M/s)   Per Row(ns)   Relative
+        ------------------------------------------------------------------------------------------------
+        Parquet Vectorized                            8088 / 8297          0.1        7713.2       1.0X
+        Parquet Vectorized (Pushdown)                 7110 / 8674          0.1        6780.8       1.1X
```

Comment:
The difference between the best and the avg is big. We need to increase `minNumIters`

Reply:
I'll increase from 2 to 5.